__tests__/api.test.js

```diff
@@ -12,6 +13,18 @@ afterAll(() => {
   connection.end();
 });
 
+describe("/api - GET request", () => {
+  it("responds with a status code of 200 and a JSON describing all the available endpoints on the api", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .expect("Content-Type", "application/json; charset=utf-8")
+      .then((response) => {
+        expect(JSON.parse(response.body.endpoints)).toEqual(endpointsDocument);
+      });
```

Comment:
Great work. You might want to me more specific by checking the shape of each object with matchers like [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)